src/mixins/filing-template-mixin.ts

```diff
@@ -180,15 +179,19 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
 
     // add in Registration data
     if (this.isCorrectedRegistration) {
-      filing.correction.business.naicsCode = this.getCurrentNaics.naicsCode
-      filing.correction.business.naicsDescription = this.getCurrentNaics.naicsDescription
+      filing.correction.business = {
+        naicsCode: this.getCurrentNaics.naicsCode,
+        naicsDescription: this.getCurrentNaics.naicsDescription
+      }
```

Comment:
Bugfix: could not assign `naicsCode` or `naicsDescription` because filing.correction.business did not exist.